cpu/esp32/doc.md

```diff
@@ -535,12 +535,16 @@ if you have already used ESP-IDF directly.
 
 Once the ESP32x tools are installed in the directory specified by the
 environment variable `$IDF_TOOLS_PATH`, the shell script
-`$RIOTBASE/dist/tools/esptools/install.sh` can be sourced to export the
+`$RIOTBASE/dist/tools/esptools/export.sh` can be sourced to export the
 paths of the installed tools using again the environment variable
 `$IDF_TOOLS_PATH`.
+By writing the `source` command in front of the script, the script
+runs in the same environment as the current shell and sets the local
+environment variables, which would otherwise be lost on termination
+of the script.
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-$ . dist/tools/esptools/export.sh
+$ source dist/tools/esptools/export.sh
```

Comment:
While the `.` is part of the POSIX Spec, the `source` command isn't and therefore not supported by all shells. It is not supported by the `dash` which is used in scripts as `/bin/sh`. That is, changing it to the `source` is not a good idea.
```suggestion
$ . dist/tools/esptools/export.sh
```